mods/map/init.lua

```diff
@@ -0,0 +1,66 @@
+map = {}
+
+
+-- Global function to allow overriding
+
+function map.check_for_map(player)
+	if creative.is_enabled_for(player:get_player_name()) or
```

Comment:
I suggest making this an optional dependency

Reply:
Done and tested without creative mod.